INSTALL.md

```diff
@@ -134,14 +173,45 @@ the repository clone in a directory called `aseprite`):
 
 # Mac OS X details
 
-From v1.1.4 we compile with Mac OS X 10.11 SDK universal. You should
-run cmake with the following parameters:
+From v1.1.4 we compile with Mac OS X 10.11 SDK universal and skia.
+
+See the [skia instructions](#building-skia-dependency-os-xlinuxbsd)
+from how to build skia.
+
+You should run cmake with the following parameters:
+
+    cd aseprite
+    mkdir build
+    cd build
+
+With Ninja:
+
+```
+cmake -D "CMAKE_OSX_ARCHITECTURES:STRING=x86_64" -D "CMAKE_OSX_DEPLOYMENT_TARGET:STRING=10.7" \
+-DUSE_ALLEG4_BACKEND=OFF -DUSE_SKIA_BACKEND=ON \
+-DSKIA_DIR=PATH/TO/SKIA \
+-D "CMAKE_OSX_SYSROOT:PATH=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk" \
+-D "WITH_HarfBuzz:BOOL=OFF" \
+-GNinja ..
+
+ninja
+```
+
+with normal make
+
+```
+cmake -D "CMAKE_OSX_ARCHITECTURES:STRING=x86_64" -D "CMAKE_OSX_DEPLOYMENT_TARGET:STRING=10.7" \
+-DUSE_ALLEG4_BACKEND=OFF -DUSE_SKIA_BACKEND=ON \
+-DSKIA_DIR=PATH/TO/SKIA \
+-D "CMAKE_OSX_SYSROOT:PATH=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk" \
+-D "WITH_HarfBuzz:BOOL=OFF" ..
+
+make
+```
+
+run
 
-    -D "CMAKE_OSX_ARCHITECTURES:STRING=x86_64"
-    -D "CMAKE_OSX_DEPLOYMENT_TARGET:STRING=10.7"
-    -D "CMAKE_OSX_SYSROOT:PATH=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk"
-    -D "WITH_HarfBuzz:BOOL=OFF"
-    -D "SKIA_DIR:PATH=/directory-which-contains-a-compiled-skia"
+`bin/aseprite`
 
```

Comment:
I will consolidate this section.

I don't like the noise either - I'd rather keep it brief (because adding this also means there's more to update if a detail changes). I'm trying to balance that and potential ambiguity for newcomers - some of which are not familiar with build systems like CMake.